app/script/view_model/MainViewModel.js

```diff
@@ -83,6 +83,7 @@ z.viewModel.MainViewModel = class MainViewModel {
     this.loading = new z.viewModel.LoadingViewModel(this, repositories);
     this.shortcuts = new z.viewModel.ShortcutsViewModel(this, repositories);
     this.title = new z.viewModel.WindowTitleViewModel(this, repositories);
+    this.favicon = new z.viewModel.FaviconViewModel(window.amplify);
```

Comment:
Thing to note here, I decided to inject amplify as a dependency of the class so that we slowly stop rely on things being published on `window`